inloop/solutions/views.py

```diff
@@ -150,7 +154,10 @@ def post(self, request, slug):
             return redirect_to_upload
 
         if not all([f.name.endswith(".java") for f in uploads]):
```

Comment:
Forgot to copy & paste? (good example why code duplication is evil – will create an issue for that).